javascript/URDFLoader.js

```diff
@@ -143,6 +146,38 @@ class URDFLoader {
     }
 
     /* Private Functions */
+
+    // Resolves the path of mesh files
+    _resolveMeshPath(pkg, meshPath){
+
+        // Remove "package://" keyword and split meshPath at the first slash
+        const [targetPkg, relPath] = meshPath.replace(/^package:\/\//, '').split(/\/(.+)/);
+
+        if (typeof pkg === 'string') {
+            // "pkg" is one single package
+
+            if (pkg.endsWith(targetPkg)) {
```

Comment:
What case is this handling? I would expect this to implement what was happening previously, which was just `pkg + '/' + filename`. Is there something that didn't seem to be working?

Admittedly it was a bit of a hack previously because it ignored the way ROS packages were supposed to work. I may rethink this part, now.

Reply:
Sounds good! Thanks for the explanation.